lib/msal-browser/src/internals.ts

```diff
@@ -8,11 +8,10 @@
  * Breaking changes to these APIs will be shipped under a minor version, instead of a major version.
  */
 
-// Cache Manager
+// Cache
 export { BrowserCacheManager } from "./cache/BrowserCacheManager";
-
-// Cache entities
 export { CacheRecord } from "./cache/entities/CacheRecord";
+export { ICachePlugin, ISerializableTokenCache, TokenCacheContext, CacheManager } from "@azure/msal-common";
```

Comment:
They are used by 1p `BrokerResponseHandler`

Reply:
They shouldn't be - I'd prefer to do the right thing here and remove the unneeded imports from the 1P lib rather than exporting additional things from msal-browser